### PR TITLE
add role

### PR DESCRIPTION
**Pull request by southeo**

### CODE REVIEW

File: test/data-exporter-backend/data-exporter-backend-roles.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  }
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: data-export-job
+  name: data-exporter-backend-role
+rules:
+  - apiGroups: ["apps"] # "" indicates the core API group
+    resources: ["deployments"]
```
Comment (samleeflang) [inline test/data-exporter-backend/data-exporter-backend-roles.yaml:16]: We deploy the export-jobs as k8s jobs right? Then this need to change to jobs

Comment (samleeflang) [reply test/data-exporter-backend/data-exporter-backend-roles.yaml:16]: I believe it is apiGroups: `batch` resources: `jobs`

File: test/data-exporter-backend/data-exporter-backend-roles.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```
Comment (samleeflang) [inline test/data-exporter-backend/data-exporter-backend-roles.yaml:17]: Do you need all of these, I don't think it updates, patches, watches or delete jobs right?

File: test/data-exporter-backend/data-exporter-backend-roles.yaml (hunk 1 above)
Comment (southeo) [reply test/data-exporter-backend/data-exporter-backend-roles.yaml]: fixed

**Review by samleeflang — APPROVED** [review]
:mortar_board:

**Pull request merged**